test/e2e/util/k8s/common.go

```diff
@@ -77,3 +78,61 @@ func WaitForPods(ctx context.Context, client TestClient, namespace string, pods
 	}
 	return nil
 }
+
+func GetPvcByPodName(ctx context.Context, namespace, podName string) ([]string, error) {
```

Comment:
Can we implement the same functionality by using the Kubernetes Go client instead so that the cases will not dependent on the `grep`, `awk`, ... binaries?